sketch.js

```diff
@@ -0,0 +1,39 @@
+
+require(["p5"], function (p5) {
+    var myp5 = new p5(function (sketch) {
+        sketch.disableFriendlyErrors = true
+        sketch.setup = function () {
+            var canvas = sketch.createCanvas(400, 400)
+            canvas.parent('sketch-div')
+        }
+
+        sketch.draw = function () {
+            with (sketch) {
```

Comment:
using with is not recommended you can use es6 destructuring assignment
const { background, push, translate. .. } = sketch